src/main/java/by/home/controller/Direction.java

```diff
@@ -0,0 +1,33 @@
+package by.home.controller;
+
+import by.home.entity.Circle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Direction {
+
+    private DataReader dataReader;
+    private CircleValidator circleValidator;
+    private CircleCreator circleCreator;
+
+    public Direction() {
```

Comment:
public Director( DataReader dataReader, CircleValidator circleValidator, CircleCreator circleCreator) {
this.dataReader = dataReader;
this.circleValidator = circleValidator;
this.circleCreator = circleCreator;
}